src/main/python/rerank/scripts/utils.py

```diff
@@ -0,0 +1,116 @@
+import shlex
+import subprocess
+import sys
+import string
+printable = set(string.printable)
+printable.remove("\n")
+printable.remove("\t")
+printable.remove("\r")
+import os
+
+import jnius_config
+jnius_config.set_classpath("target/anserini-0.4.1-SNAPSHOT-fatjar.jar")
+from jnius import autoclass
+JString = autoclass('java.lang.String')
+JSearcher = autoclass('io.anserini.search.SimpleSearcher')
+from nltk.tokenize import TweetTokenizer
+tknzr = TweetTokenizer()
+
+def cal_score(fn_qrels="src/main/resources/topics-and-qrels/qrels.microblog2014.txt", prediction="score.txt"):
+    cmd = "./etc/trec_eval.9.0/trec_eval {judgement} {output} -m map -m recip_rank".format(output=prediction, judgement=fn_qrels)
```

Comment:
If this script is meant to be run from Anserini base directory, then the relative path should be `eval/trec_eval.9.0/trec_eval`, right?

Reply:
Right. I have accidentally rename it because of historical reason.